src/Conference.tsx

```diff
@@ -408,22 +505,28 @@ const Conference: React.FC = (props) => {
     };
   }, [immersiveMode]);
 
-  return (
-    <div
-      className={immersiveMode ? classes.conferenceNoCursor : classes.conference}
-      onMouseMove={() => setImmersiveMode(false)}
-    >
-      <SessionInfo
-        immersiveMode={immersiveMode}
-        userAgent={userAgent}
-        conferenceId={conferenceId}
-        conferenceCode={conferenceCode}
-        conferencePhoneNumber={phoneNumber}
-        participantId={participantId}
-        localStream={localStream}
-        remoteStreams={Object.values(remoteStreams)}
-        error={error}
-      />
+  if (displayDialog) {
```

Comment:
We should try replacing this with testing whether the deviceSelectionData is set. If it works then we can get rid of the displayDialog state variable.